src/WinXP/apps/DisplayProperties/ScreenSaverTab.js

```diff
@@ -1,32 +1,132 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import styled from 'styled-components';
 
-import display from '../../../assets/properties/displayProperties/display.png';
+import display from 'assets/properties/displayProperties/display.png';
+import arrowDown from 'assets/properties/displayProperties/icons/arrowDown.png';
+import Button from 'components/Button';
+import NumberInput from 'components/NumberInput';
+import CheckBox from 'components/CheckBox';
+import ScreenSaver from 'components/ScreenSavers';
+import { SCREEN_SAVER } from './utils';
+import { SCREEN_SAVER_PREVIEW } from 'WinXP/constants/actions';
+
+function ScreenSaverTab({ state, dispatch, appContext }) {
+  const { value, wait } = state.displayProperties.screenSaver;
+  const [screenSaverState, setScreenSaverState] = useState({ value, wait });
+  const [isNone, setIsNone] = useState(false);
+
+  useEffect(() => {
+    dispatch({ type: SCREEN_SAVER, payload: screenSaverState });
+  }, [screenSaverState, dispatch]);
+
+  useEffect(() => {
+    setIsNone(screenSaverState.value === '(None)' ? true : false);
```

Comment:
No need for the ternary, the comparison already returns a boolean
```suggestion
    setIsNone(screenSaverState.value === '(None)';
```